.github/workflows/integration_test_charm.yaml

```diff
@@ -362,6 +367,15 @@ jobs:
           sg '${{ steps.parse-cloud.outputs.group }}' -c "microk8s.kubectl rollout status --namespace kube-system --watch --timeout=5m deployments/coredns"
           sg '${{ steps.parse-cloud.outputs.group }}' -c "retry --times 3 --delay 5 -- sudo microk8s enable hostpath-storage"
           sg '${{ steps.parse-cloud.outputs.group }}' -c "microk8s.kubectl rollout status --namespace kube-system --watch --timeout=5m deployments/hostpath-provisioner"
+          if [[ '${{ inputs.metallb-addon }}' = 'true' ]]
+          then
+            if [[ '${{ inputs.architecture }}' == 'arm64' ]]
```

Comment:
why `jq` is not installed on arm?

Reply:
not entirely sure, but tests on arm64 failed due to lack of jq here. would you suggest another manner of installation?

Reply:
I would install jq on arm by default (re-checked deb packages on jammy and snap, both available for arm and amd).

Reply:
installed by default in [0f3b66b](https://github.com/canonical/data-platform-workflows/pull/244/commits/0f3b66b38b236255d6856e52c5104b534062dded)